### PR TITLE
Run one websockify test on macOS during CI. NFC

### PR DESCRIPTION
**Pull request by sbc100**
This requires more consistent running of `pip install`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -150,49 +154,33 @@ commands:
...
-            HOMEBREW_NO_AUTO_UPDATE: "1"
-          command: |
-            brew list cmake || brew install cmake
-            brew list python3 || brew install python3
       - run:
           name: run tests
           command: |
             ./tests/runner << parameters.test_targets >>
             $EMSDK_PYTHON ./tools/check_clean.py
+  run-tests-linux:
```
Comment (kripken) [inline .circleci/config.yml:170]: why is this called linux? looks like it is very generic, just prepares and then runs tests?

does this overlap with another PR? the `prepare-for-tests` part looks familiar...

Comment (sbc100) [reply .circleci/config.yml:170]: Yes I have other PR that was trying to do a similar thing on windows: #16378.

But I figured making it work on mac first was a good plan.

Comment (tlively) [reply .circleci/config.yml:170]: > But I figured making it work on mac first was a good plan.

Do you mean Linux?

**Review by tlively — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (tlively) [reply .circleci/config.yml:170]: Oh, I see. This differentiates the refactored linux steps from the new mac steps.

Comment (sbc100) [reply .circleci/config.yml:170]: No, we already run all the socket tests on linux.. this change just adds the running on one of them on macOS

**Pull request merged**